BootloaderCommonPkg/Library/ElfLib/Elf64Lib.c

```diff
@@ -87,7 +87,7 @@ RelocateElf64Sections  (
     CurPtr  = CurPtr + Elf64Hdr->e_shentsize;
     if ((Rel64Shdr->sh_type == SHT_REL) || (Rel64Shdr->sh_type == SHT_RELA)) {
       Sec64Shdr = GetElf64SectionByIndex (ElfCt->FileBase, Rel64Shdr->sh_info);
-      if (!IsTextShdr(Sec64Shdr) && !IsDataShdr(Sec64Shdr)) {
+      if ((Sec64Shdr != NULL) && !IsTextShdr(Sec64Shdr) && !IsDataShdr(Sec64Shdr)) {

```

Comment:
If Sec64Shdr returns NULL,  you code flow will run line #94.
I guess better to add 
if (Sec64Shdr == NULL) {
  continue;
}